packages/mux-player/src/styles.css

```diff
@@ -21,9 +18,6 @@ media-controller {
   --media-control-hover-background: transparent;
   --media-range-track-background-color: rgba(255, 255, 255, 0.5);
   --media-range-track-border-radius: 3px;
-  --media-aspect-ratio: auto;
-  display: block;
-  width: 100%;
```

Comment:
width is still needed as well

Reply:
added, I didn't reproduce anything that required it yet or broke without it but it does make sense to have `width: 100%` as well

Reply:
I was seeing some issues particularly with portrait videos and trying to set the width, which this resolved.